etc/client.api.report.api.md

```diff
@@ -475,6 +475,10 @@ export type ObjectMetadata = {
 
 // @public (undocumented)
 export interface ObjectSet<Q extends ObjectOrInterfaceDefinition = any, _UNUSED = any> extends MinimalObjectSet<Q> {
+    // Warning: (ae-forgotten-export) The symbol "EXPERIMENTAL_ObjectSetListener" needs to be exported by the entry point index.d.ts
+    //
+    // @alpha
+    readonly [__EXPERIMENTAL__NOT_SUPPORTED_YET_subscribe]: (listener: EXPERIMENTAL_ObjectSetListener<Q>) => () => unknown;
```

Comment:
Any reason you didn't just want to call these all `__EXPERIMENTAL_subscribe` etc.?

Reply:
scarier the better! Also we can rename it as much as we want so if you like it smaller we can do that later.